MAUI/DataGrid/selection.md

```diff
@@ -556,4 +556,108 @@ By default, the keyboard navigation will be enabled when setting the selection a
                     ItemsSource="{Binding OrderInfoCollection}" >
 </syncfusion:SfDataGrid>
 {% endhighlight %}
+{% endtabs %}
+
+
+## Customizing the selection controller behavior
+
+For customizing the selection controller behavior, initially create the instance for selection controller then, we have to create a class by inheriting the `DataGridRowSelectionController`. And then override the the methods you want to customize.
```

Comment:
The default keyboard selection behaviors can be customized by setting the instance of the custom `DataGridRowSelectionController` to the `SfDataGrid.SelectionController` property.

Reply:
Changed